automata/fa/dfa.py

```diff
@@ -499,28 +499,47 @@ def _minify(
         If the input DFA is partial, then the result is also a partial DFA
         """
 
-        # First, assemble backmap and equivalence class data structure
-        eq_classes = PartitionRefinement(reachable_states)
-        refinement = eq_classes.refine(reachable_final_states)
-
-        final_states_id = (
-            refinement[0][0] if refinement else next(iter(eq_classes.get_set_ids()))
-        )
+        reachable_states = set(reachable_states)
 
         # Per input-symbol backmap (tgt -> origin states)
         transition_back_map: Dict[str, Dict[DFAStateT, List[DFAStateT]]] = {
-            symbol: {end_state: list() for end_state in reachable_states}
+            symbol: {end_state: [] for end_state in reachable_states}
             for symbol in input_symbols
         }
 
+        trap_state = None
+
         for start_state, path in transitions.items():
             if start_state in reachable_states:
-                for symbol, end_state in path.items():
-                    symbol_dict = transition_back_map[symbol]
-                    # If statement here needed to ignore certain transitions
-                    # when minifying a partial DFA.
-                    if end_state in symbol_dict:
-                        symbol_dict[end_state].append(start_state)
+                for symbol in input_symbols:
+                    end_state = path.get(symbol)
+                    if end_state is not None:
+                        # for symbol, end_state in path.items():
```

Comment:
[minor] remove this comment

Reply:
Removed here: https://github.com/caleb531/automata/pull/184/commits/ecd87c4e0350cf01e2d721ba5c61b7e9cd2a39f1